README.md

```diff
@@ -1 +1,62 @@
-# mag-calculator
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: Bookstore
+
+> Description the project: This project is for Magiciant Calculator. ;)
+
+
+## Built With
+
+- Major languages
+> Html
+> CSS
+> JavaScript
+> React
+
+
+## Authors
+
+👤 **Sajjad Rajaee**
+
+- GitHub: [@sajjadrajaee](https://github.com/sajjadrajaee)
+- Twitter: [@sajjadrajaee](https://twitter.com/sajjadrajaee)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sajjadrajaee)
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you shift the `## Authors` section after the `## Getting Started` section to match the **[Micoverse README template](https://github.com/microverseinc/readme-template)**.